helm/rook-operator/Chart.yaml

```diff
@@ -5,6 +5,8 @@ dependencies:
     version: 0.2.0
   - name: rook-ceph-cluster
     version: 0.0.1
+  - name: library
+    version: "0.0.1"
```

Comment:
This chart doesn't exist yet as it will need to be added in a separate subtree merge PR